infrastructure/modules/migration/envs/dev/main.tf

```diff
@@ -0,0 +1,58 @@
+module "database_migration_service" {
+  source  = "terraform-aws-modules/dms/aws"
+  version = "~> 2.0"
+
+  # Subnet group
+  repl_subnet_group_name        = module.networking.private_subnet_group_name # check is this correct?
+  repl_subnet_group_description = "The name of the subnet group used with the databases"
+  repl_subnet_group_subnet_ids  = module.networking.private_subnet_ids
+
+  endpoints = {
+    source = {
+      database_name               = "npd_etl"
+      endpoint_id                 = "npd-east-dev-etl-db"
+      endpoint_type               = "source"
+      engine_name                 = "postgresql"
+      username                    = "npd_etl"
+      password                    = "youShouldPickABetterPassword123!" # where to get this?
+      port                        = 5432
+      server_name                 = "npd-east-dev-etl-db.cc3asw0omsvl.us-east-1.rds.amazonaws.com" # can this be a reference instead of hard coded?
+      ssl_mode                    = "require"
+    }
+
+    destination = {
+      database_name = "npd"
+      endpoint_id   = "npd-east-dev-fhir-api-db"
+      endpoint_type = "target"
+      engine_name   = "postgresql"
+      username      = "npd"
+      password      = "passwordsDoNotNeedToMatch789?" # where to get this?
+      port          = 5432
+      server_name   = "npd-east-dev-fhir-api-db.cc3asw0omsvl.us-east-1.rds.amazonaws.com" # can this be a reference instead of hard coded?
+      ssl_mode      = "require"
+    }
+  }
```

Comment:
because we're connecting to two databases, you'll need to update `variables.tf` to accept information about both databases.